src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -901,6 +902,7 @@ export default class UniversalDiscoveryModule extends Component {
             bookmarksPerPage,
             restInfo,
             selectedContent,
+            allowedContentTypes: cotfAllowedContentTypes,
```

Comment:
Here, the `allowedContentTypes` should be passed not `cotfAllowedContentTypes`.

Reply:
in HTML attribute `data-udw-config` allowed content types are in `ctofAllowedContentTypes` (look in this [prtcsr](https://user-images.githubusercontent.com/5550410/69718254-2f5c1680-110e-11ea-90ce-e5d89e142ed9.png)) and in line you commented I map value of this key to `allowedContentTypes` key and next pass to `SearchPanelComponent` with `allowedContentTypes` key (not `ctofAllowedContentTypes`) (look in `search.component.js` line 45: `query.ContentTypeIdentifierCriterion = this.props.allowedContentTypes;`)

Reply:
This key is only for content on the fly, all other tabs should use the `allowedContentTypes`. I know this might be confusing but it was done for the BC.